Updating the suggested flow for updating a table (#1709)
ClickHouse/clickhouse-docs: docs/en/integrations/data-ingestion/kafka/kafka-table-engine.md

Either place the above snippet inside a new file under your conf.d/ directory or merge it into existing configuration files. For settings that can be configured, see [here](../../../engines/table-engines/integrations/kafka.md#configuration).

> We're also going to create a database called `KafkaEngine` to use in this tutorial.

Reviewer: ```suggestion
We're also going to create a database called `KafkaEngine` to use in this tutorial:
```

##### 4. Create and populate the topic

> Next, we're going to create a topic. 
> There are several tools that we can use to do this.
> If we're running Kafka locally on our machine or inside a Docker container, [rpk](https://docs.redpanda.com/current/get-started/rpk-install/) works well. 
> We can create a topic called `github` with 5 partitions by running the following command:

Reviewer: ```suggestion
Next, we're going to create a topic. There are several tools that we can use to do this. If we're running Kafka locally on our machine or inside a Docker container, [RPK](https://docs.redpanda.com/current/get-started/rpk-install/) works well. We can create a topic called `github` with 5 partitions by running the following command:
```

```bash
rpk topic create -p 5 github --brokers <host>:<port>
```

If we're running Kafka on the Confluent Cloud, we might prefer to use the [Confluent CLI](https://docs.confluent.io/platform/current/tutorials/examples/clients/docs/kcat.html#produce-records):

```bash
confluent kafka topic create --if-not-exists github
```

> Now we need to populate this topic with some data, which we'll do using [kcat](https://github.com/edenhill/kcat).
> We can run a command similar to the following if we're running Kafka locally with authentication disabled:

Reviewer: ```suggestion
Now we need to populate this topic with some data, which we'll do using [kcat](https://github.com/edenhill/kcat). We can run a command similar to the following if we're running Kafka locally with authentication disabled:
```

> ```bash

Reviewer: ```suggestion
```

```bash
cat github_all_columns.ndjson | 
kcat -P \
  -b <host>:<port> \
  -t github
```

Or the following if our Kafka cluster uses SASL to authenticate:

```bash
cat github_all_columns.ndjson | 
kcat -P \
  -b <host>:<port> \
  -t github
  -X security.protocol=sasl_ssl \
  -X sasl.mechanisms=PLAIN \
  -X sasl.username=<username>  \
  -X sasl.password=<password> \
```

The dataset contains 200,000 rows, so hopefully it should be ingested in just a few seconds.
If you want to work with a larger dataset, take a look at [the large datasets section](https://github.com/ClickHouse/kafka-samples/tree/main/producer#large-datasets) of the [ClickHouse/kafka-samples](https://github.com/ClickHouse/kafka-samples) GitHub repository.

##### 5. Create the Kafka table engine

> The below example creates a table engine with the same schema as the merge tree table. Note that this isn’t required, e.g. you can have an alias or ephemeral columns in the target table. The settings are important; however - note the use of `JSONEachRow`` as the data type for consuming JSON from a Kafka topic. The values “github” and “clickhouse” represent the name of the topic and consumer group names, respectively. Note that the topics can actually be a list of values.

Reviewer: ```suggestion
The below example creates a table engine with the same schema as the merge tree table. This isn’t strickly required, as you can have an alias or ephemeral columns in the target table. The settings are important; however - note the use of `JSONEachRow` as the data type for consuming JSON from a Kafka topic. The values `github` and `clickhouse` represent the name of the topic and consumer group names, respectively. The topics can actually be a list of values.
```

##### Stopping & restarting message consumption

To stop message consumption, you can detach the Kafka engine table:

```sql
DETACH TABLE github_queue;
```

This will not impact the offsets of the consumer group. To restart consumption, and continue from the previous offset, reattach the table.

```sql
ATTACH TABLE github_queue;
```

##### Adding Kafka Metadata

> It can be useful to keep track of the metadata from the original Kafka messages after it's been ingested into ClickHouse.
> For example, we may want to know how much of a specific topic or partition we have consumed. For this purpose, the Kafka table engine exposes several [virtual columns](../../../engines/table-engines/index.md#table_engines-virtual_columns). These can be persisted as columns in our target table by modifying our schema and materialized view’s select statement.

Reviewer: ```suggestion
It can be useful to keep track of the metadata from the original Kafka messages after it's been ingested into ClickHouse. For example, we may want to know how much of a specific topic or partition we have consumed. For this purpose, the Kafka table engine exposes several [virtual columns](../../../engines/table-engines/index.md#table_engines-virtual_columns). These can be persisted as columns in our target table by modifying our schema and materialized view’s select statement.
```
